src/features/collaborator-avatars/CollaboratorList.scss

```diff
@@ -1,6 +1,9 @@
 @import '../../styles/variables';
 
 .usm-collaborator-list {
+    display: flex;
+    flex-direction: column;
```

Comment:
For some reasons sometimes this view might have been broken so this aims to fix that:

<img width="1723" alt="Screenshot 2025-04-20 at 12 37 03" src="https://github.com/user-attachments/assets/dc9ed13e-62ab-42b8-81e1-36cb500e1f8d" />